upload_stage_rpms

```diff
@@ -5,4 +5,8 @@
 USER='yumrepostage'
 HOST='web01.osuosl.theforeman.org'
 
-rsync --checksum --perms --recursive --links --verbose --partial --one-file-system --delete-after "tmp/$PROJECT/$VERSION" "$USER@$HOST:rsync_cache/$PROJECT"
+for os in $OSES; do
+  BASE="tmp/$PROJECT/$VERSION/$os"
+
+  rsync --checksum --perms --recursive --links --verbose --partial --one-file-system --delete-after "$BASE" "$USER@$HOST:rsync_cache/$PROJECT/$VERSION"
```

Comment:
Will this sync `$os` or all the contents of `$os` into `$USER@$HOST:rsync_cache/$PROJECT/$VERSION`? I'm always unsure.

If all the OSes are already in `tmp/$PROJECT/$VERSION` then I'd expect the old script to do the right thing.

Reply:
I think you are asking for this:

```
[2023-11-20T16:16:20.417Z] + export 'RSYNC_RSH=ssh -i /home/jenkins/workspace/staging_key/rsync_yumrepostage_key'

[2023-11-20T16:16:20.417Z] + RSYNC_RSH='ssh -i /home/jenkins/workspace/staging_key/rsync_yumrepostage_key'

[2023-11-20T16:16:20.417Z] + /usr/bin/rsync --checksum --perms --recursive --links --verbose --partial --one-file-system --delete-after tmp/pulpcore/nightly/el8 yumrepostage@web01.osuosl.theforeman.org:rsync_cache/pulpcore/nightly

[2023-11-20T16:16:22.135Z] building file list ... done

[2023-11-20T16:16:22.135Z] el8/

[2023-11-20T16:16:22.740Z] el8/source/

[2023-11-20T16:16:23.045Z] el8/source/repodata/

[2023-11-20T16:16:23.045Z] el8/source/repodata/207c4eab8244c17a7fc812d4051d29a45f7bffe4b9d9c9a04bfb39a298ee4e5c-primary.xml.gz

[2023-11-20T16:16:23.045Z] el8/source/repodata/2589bf01e306087030ff6e32b40e3954b2fab6d5b634511b5904b0de30cc5c97-other.sqlite.bz2

[2023-11-20T16:16:23.045Z] el8/source/repodata/2b933fcb65be547282b930309aac49b58d7501f50088e255e201a20d2ab328c9-filelists.sqlite.bz2

[2023-11-20T16:16:23.045Z] el8/source/repodata/d9514773b6d653af279cc7188dfe5f3990c8b99a8392866c07a59b6595a765f1-primary.sqlite.bz2

[2023-11-20T16:16:23.045Z] el8/source/repodata/repomd.xml

[2023-11-20T16:16:23.045Z] el8/x86_64/

[2023-11-20T16:16:23.247Z] el8/x86_64/repodata/

[2023-11-20T16:16:23.247Z] el8/x86_64/repodata/42d0ab6d0f55dfbefd42e4d73adf596c462e28742874eb95d2f06b34b6db3aa0-filelists.sqlite.bz2

[2023-11-20T16:16:23.247Z] el8/x86_64/repodata/7e8f726ca0bd46125ef0487ca8e8e827c71e86c01cc70bd8060cc0f236389c73-primary.sqlite.bz2

[2023-11-20T16:16:23.247Z] el8/x86_64/repodata/c9fe2ebc7c7ba7e1e37a2af806bd07d9154d7ceb95f7f55dabd4602084a4c802-other.sqlite.bz2

[2023-11-20T16:16:23.247Z] el8/x86_64/repodata/da41e82727fb5a2076105c0b6ec0963e2061bba7203029ec480bacfefccdc5e9-primary.xml.gz

[2023-11-20T16:16:23.247Z] el8/x86_64/repodata/repomd.xml

[2023-11-20T16:16:23.247Z] deleting el8/source/repodata/f933ee25994caa93c907bef06abace4d93dd3092c6e43ae4c10402910eb4a60b-other.sqlite.bz2

[2023-11-20T16:16:23.247Z] deleting el8/source/repodata/dbb296c2fd151643f2f00abcce25f7e0ba5d20076fde5190fd7d6590e3833b81-primary.sqlite.bz2

[2023-11-20T16:16:23.247Z] deleting el8/source/repodata/cc6daa6fd5a72b6204245013522951a6ad5ba5fff2e5b3c121addc64eb401509-filelists.sqlite.bz2

[2023-11-20T16:16:23.247Z] deleting el8/source/repodata/bfa246a0c8aef4dad7465f24ec26f624c46ac09252150c931c05762ba902b954-primary.xml.gz

[2023-11-20T16:16:23.247Z] deleting el8/x86_64/repodata/efd132ed558499d10bcc60212e54b3057535bd8f86313ae504bd51b2ea124f6b-primary.sqlite.bz2

[2023-11-20T16:16:23.247Z] deleting el8/x86_64/repodata/ca81eed49109b8c21633ffe640a8f908488bd49c1ea170e32cfdfefa3a413d3b-primary.xml.gz

[2023-11-20T16:16:23.247Z] deleting el8/x86_64/repodata/ac1922f7f27481802f34c68d36b80f6caeccd65cda805dfaf9ca2cb9ee26bbb4-other.sqlite.bz2

[2023-11-20T16:16:23.247Z] deleting el8/x86_64/repodata/1b022db2ac763010cd9d9b974b4c53a3553ec52b85169229739cc24863cd9815-filelists.sqlite.bz2
```